packages/twenty-front/src/modules/settings/data-model/fields/forms/components/SettingsDataModelFieldSettingsFormCard.tsx

```diff
@@ -73,6 +73,7 @@ const previewableTypes = [
   FieldMetadataType.Text,
   FieldMetadataType.Address,
   FieldMetadataType.RawJson,
+  FieldMetadataType.Domain,
```

Comment:
Let's keep it behind a feature flag maybe if additionalX is not even displayed?
The goal would be to do a perfect "primary + additionalXXX" composite field (we can start with link, then phone number, emails)